### PR TITLE
Add message when pint is missing and instructions for users.

### PR DESCRIPTION
**Pull request by jullit31**
This is my very first pull request, so I hope what I am doing here is correct.
This should help with #2, as well as give instructions to the user un how to use the plugin.

### CODE REVIEW

**Comment by rpalo** [comment]
Thank you for your pull request!  Congratulations!  Let me take a look at it and I'll have more feedback for you.  :smile:

File: src/main.py [hunk 1]
```diff
@@ -1,24 +1,45 @@
 """Wox Plugin class for unit conversions"""
-
-import pint
+import pkgutil
 from wox import Wox
+if pkgutil.find_loader("pint"): #Check for pint
```
Comment (rpalo) [inline src/main.py]: What do you think about something like this:

```python
try:
    import pint
    ureg = pint.UnitRegistry()
    ureg.default_format = ".5nP"
except ImportError:
    error_msg = "This plugin requieres the python module 'pint'. Run 'pip install pint' to install pint."
```

This way seems a little more pythonic to me, and avoids using a flag variable `dependencies_exist`.  Now you can either check to see if ureg exists...

```python
if not ureg:
    # stuff
```
Or you can see if there's an error message.

```python
if error_message:
    # stuff
```

That should come in handy in the lines below.  What do you think?

**Review by rpalo — COMMENTED** [review]
Thanks again for opening a pull request to help out!  I have one thought for a way to make things a little more pythonic.  Let me know if you agree.

File: src/main.py (hunk 1 above)
Comment (jullit31) [reply src/main.py]: Yes, this does look more elegant. But why the `error_msg` variable? We could just
```python
except ImportError:
   pass
```
and then `if not ureg:`. That way all messages to the user would stay inside the `query`-function.

Comment (rpalo) [reply src/main.py]: Yeah, that'll work too.  I'm cool with that.

File: src/main.py [hunk 2]
```diff
@@ -1,24 +1,43 @@
...
-        except Exception as e:
-            answer = f"Something failed. {e}"
-        result = [{
-            "Title": f"{target.title()}",
-            "SubTitle": f"{answer}".title(),
-            "IcoPath": "Images\\app.png",
-        }]
+        """Returns instructions for the user or the results of the user's search string"""
+        result = []
+        if not 'ureg' in globals():
```
Comment (jullit31) [inline src/main.py]: Has to be done like this, just `if not ureg:` results in a `NameError`.

File: src/main.py [hunk 3]
```diff
@@ -1,13 +1,11 @@
...
-if pkgutil.find_loader("pint"): #Check for pint
+try:
     import pint
     ureg = pint.UnitRegistry()
     ureg.default_format = ".5nP"
-    dependecies_exist = True
-else:
-    dependecies_exist = False
+except ImportError:
+    pass
```
Comment (rpalo) [inline src/main.py]: How about putting `ureg = None` inside this except clause?

Comment (jullit31) [reply src/main.py]: I don't see why not. Would you consider that more pythonic? (I'm actually more used to Ruby than Python; so maybe that's why I wrote it like this, as there is a nice way of checking if a variable is defined in Ruby. 😬)

Comment (rpalo) [reply src/main.py]: Yeah, usually in Python, at least from what I've seen, checking for variable existence or validity using `if var:` or `if not var:` is kind of the way to go.  You could also maybe argue that it's good not to be checking global scope for things, but allowing the nested scopes to do their thing.

That being said, there's probably not a whole lot of difference between the two methods, especially at this point.  Let me know if you want to change it or not.  If not, I'll just merge it as is!

**Comment by rpalo** [comment]
Merged!  Thanks for your help!

**Pull request merged**